.github/workflows/build_package.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches: "**"
+jobs:
+  release:
+    name: Release on GitHub and PGXN
+    runs-on: ubuntu-latest
+    container: pgxn/pgxn-tools
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Bundle the Release
+        id: bundle
```

Comment:
Why do we need to supply an `id` field here? It appears unused.

Reply:
Fixed